agnpy/synchrotron/proton_synchrotron.py

```diff
@@ -0,0 +1,183 @@
+# module containing the synchrotron radiative process
+import numpy as np
+import astropy.units as u
+from astropy.constants import e, h, c, m_e, m_p, sigma_T, G
```

Comment:
h, m_e, sigma_T and G are not needed.
If you use an IDE as visual studio, you can actually see which names (imports or defined variables) are not needed in the code.